scripts/ceph_s3_tests/Makefile

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+# This file will setup & configure the ceph/s3-tests repo.
+# Commands:
+#	1. make help # To check supported tools by this script.
+#	2. make setup-s3tests # Clone and setup s3-tests repo under cortx-test
+#	3. make configure-s3tests  # Generate configuration file for running tests
+# Note: Please provide existing s3account user access, secret keys through command line.
+
+CORTX_TESTS_DIR=/root/cortx-test
+ENDPOINT=s3.seagate.com

```

Comment:
This endpoint indicates if you are using /etc/hosts. Is it ?

Reply:
Specifying a default value here, but expectation is for user to specify ENDPOINT at command line if s3.seagate.com cannot be resolved